packages/web/src/components/gcds-fieldset/gcds-fieldset.tsx

```diff
@@ -29,8 +29,7 @@ import i18n from './i18n/i18n';
 @Component({
   tag: 'gcds-fieldset',
   styleUrl: 'gcds-fieldset.css',
-  shadow: false,
-  scoped: true,
+  shadow: { delegatesFocus: true },
```

Comment:
Do we need `formAssociated: true,` here? Just ignore if we don't

Reply:
No we won't since fieldset only groups form elements and doesn't submit anything